Library/Homebrew/test/Gemfile

```diff
@@ -3,5 +3,17 @@ source "https://rubygems.org"
 gem "mocha", "~> 1.1"
 gem "minitest", "~> 5.3"
 gem "rake", "~> 10.3"
-gem "simplecov", "~> 0.10.0", :require => false
-gem "coveralls", "0.8.9", :require => false
+# This is a patched version of the v0.11.1. Switch back to the stable version
+# when #436 and #437 have been merged upstream.
+# https://github.com/colszowka/simplecov/pull/436
```

Comment:
Mind accepting your own PR for that one? It has conflicts and your patch is simple enough to be a separate PR.